esp/applications/eclqueries/testauth_authorization_map.yaml

```diff
@@ -0,0 +1,34 @@
+authNZ:
+   test:
+      name: test
+      type: test
+      LibName: libtestauthSecurity.so
+      useResourceMapsFrom: ldap
+      userAccess:
+      -  userName: testadmin
+         #password: MW6vRCpKS7AhO/1YBazKOg== # optional. If missing, use the userName as the password.
```

Comment:
Probably better not to check in something that looks like a password token, even in a comment and even if it's not valid anywhere - it will raise some red flags. Something like 

    # password: put_your_test_password_here ...
    
might be better